src/status_im/hardwallet/core.cljs

```diff
@@ -1843,19 +1843,21 @@
   (let [tag-was-lost? (= "Tag was lost." (:error error))]
     (fx/merge cofx
               (if tag-was-lost?
```

Comment:
isn't it a when?

Reply:
doesn't really change here anything and I just didn't touch it :)

Reply:
fixed